website/src/pages/index.js

```diff
@@ -106,6 +106,8 @@ function Home() {
               Get Started
             </Link>
           </div>
+          <p className="hero__subtitle"></p>
+          <iframe src="https://ghbtns.com/github-btn.html?user=h2oai&repo=wave&type=star&count=true&size=large" style={{marginLeft:2+'em'}} align="middle" frameborder="0" scrolling="0" width="170" height="30" title="GitHub"></iframe>
```

Comment:
The line is way too long, can you please put every attr into a newline?

Reply:
Will do :) The marginLeft is to add spacing on left and make it center aligned as compared to Get Started button. Added a screenshot with mobile version to description